### PR TITLE
Remove Lombok

### PR DESCRIPTION
**Pull request by dturanski**

### CODE REVIEW

**Comment by paskos** [comment]
Just out of curiosity. Why removing Lombok ?

File: fleet-location-simulator/src/main/java/demo/model/PositionInfo.java [hunk 1]
```diff
@@ -12,17 +12,12 @@
  */
 package demo.model;
 
-import lombok.AccessLevel;
-import lombok.Data;
-import lombok.NoArgsConstructor;
-
 /**
- * POJO to hold position of the vehicle and some other data.
+ * POJO to hold position of the vehiclqe and some other data.
```
Comment (markfisher) [inline fleet-location-simulator/src/main/java/demo/model/PositionInfo.java]: typo

File: fleet-location-simulator/src/main/java/demo/model/ServiceLocation.java [hunk 2]
```diff
@@ -21,13 +21,11 @@
...
 import com.fasterxml.jackson.annotation.JsonUnwrapped;
 
-import lombok.Data;
-
 /**
  * @author Dave Syer
  * @author Gunnar Hillert
  */
-@Data
+
```
Comment (markfisher) [inline fleet-location-simulator/src/main/java/demo/model/ServiceLocation.java]: extra line

**Comment by dturanski** [comment]
@paskos - We're using this app as a spring-cloud reference app. For this purposes, Lombok is superfluous. Also we prefer to leave it up to the developer if they want to use it.

**Comment by golonzovsky** [comment]
-1

**Comment by markfisher** [comment]
merged as https://github.com/springone2gx2015/vehicle-fleet-demo/commit/0e7db6df68aa704248221bbf4f1a0e8a033d8c50 along with https://github.com/springone2gx2015/vehicle-fleet-demo/commit/0cd443053f41856ba4cc43b450ae2e482006d0e2

**Pull request closed without merging**